### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by arturo-seijas**
Update readme to reflect how the runner image is selected.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,9 +33,13 @@ The following workflows are available:
 | integration-test-provider | string | microk8s | Actions operator provider as defined (here)[https://github.com/charmed-kubernetes/actions-operator#usage] |
 | integration-test-series | string | '[""]' | List of series to run the tests in JSON format, i.e. '["jammy", "focal"]'. Each element will be passed to tox as --series argument |
 
+The runner image will be set to the value of `bases[0].build-on[0]` in the `charmcraft.yaml`file, defaulting to ubuntu-22.04 if the file does not exist.
```
Comment (merkata) [inline README.md:36]: needs a space before `file`, right?

**Review by varshigupta12 — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -33,9 +33,13 @@ The following workflows are available:
...
+
 * release: Promotes a charm from the selected origin channel to the selected target channel. . The following parameters are available for this workflow:
 
 | Name | Type | Default | Description |
 |--------------------|----------|--------------------|-------------------|
 | destination-channel | string | "" | Destination channel |
 | origin-channel | string | "" | Origin channel |
+
+The runner image will be set to the value of `bases[0].build-on[0]` in the `charmcraft.yaml`file, defaulting to ubuntu-22.04 if the file does not exist.
```
Comment (merkata) [inline README.md:45]: as above, missing space

File: README.md (hunk 1 above)
Comment (arturo-seijas) [reply README.md:36]: Indeed. Thx

File: README.md (hunk 2 above)
Comment (arturo-seijas) [reply README.md]: Fixed

**Review by merkata — APPROVED** [review]
Looks great!

**Review by gtrkiller — APPROVED** [review]
LGTM

**Pull request merged**